tests/Unit/L1QualificationTest.php

```diff
@@ -1,57 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace EDTF\Tests\Unit;
-
-use PHPUnit\Framework\TestCase;
-
-/**
- * Class L1DateTest
- * @covers \EDTF\Parser
- * @covers \EDTF\Interval
- * @covers \EDTF\ExtDateTime
- * @package EDTF\Tests\Unit
- */
-class L1QualificationTest extends TestCase
```

Comment:
Why did you remove these tests?

Reply:
I just want to focus on Level 0 first. I still have a copy of this tests.